etcdmgmt/etcd-mgmt.go

```diff
@@ -28,6 +64,19 @@ func StartEtcd() error {
 
 	initialAdvPeerUrls := "http://" + hostname + ":2380"
 
+	// Checking health of etcd cluster. If etcd daemon already running then
```

Comment:
Is there any specific reason for not to go for a pid check? I can understand that the same can go for a toss if an etcd instance is run independently but that should never happen IMO.